### ISSUE TITLE
Clarify healthz `unhealthy-count` and `last-unhealthy`

### ISSUE THREAD

**Issue by hellt** [openconfig/public#851]
Hi @robshakir @marcushines 

Can we clarify the difference in descriptions for `last-unhealthy` and `unhealthy-count`?
Namely, `last-unhealthy` indicates that the field should be modified any time a component is in a status that is not HEALTHY. UNSPECIFIED falls under this category.

At the same time, `unhealthy-count` only counts transitions HEALTHY<->UNHEALTHY. Will unhealthy counter increment if the component moved from HEALTHY to UNSPECIFIED?

This subtle difference may lead to differences in implementation.

https://github.com/openconfig/public/blob/de94e3b3573de05466305f8f7eb4eae960da54e7/release/models/platform/openconfig-platform-healthz.yang#L78-L122

To summarize, our reading of current model digests to the following state transition:

1. `healthy`->`unspecified` | `last-unhealthy` changes | `unhealthy-count` doesn't change
2. `unhealthy`->`unspecified` | `last-unhealthy` changes | `unhealthy-count` doesn't change
3. `unspecified`->`unhealthy` | `last-unhealthy` changes | `unhealthy-count` doesn't change
4. `unspecified`->`healthy` | `last-unhealthy` doesn't change | `unhealthy-count` doesn't change
5. `healthy`->`unhealthy` | `last-unhealthy` changes | `unhealthy-count` increments change
6. `unhealthy`->`healthy` | `last-unhealthy` doesn't change | `unhealthy-count` doesn't change

**Comment by hellt** [comment]
@marcushines then this addition should be valid - https://github.com/openconfig/public/pull/853. If I understood your comment correclty.

**Comment by dplore** [comment]
Fixed by #853
